README.md

```diff
@@ -226,6 +230,10 @@ The office is a great place to work from, but occasionally we get together and w
 
 We use Officevibe to measure employee happiness and satisfaction at work. Through this platform you can express your opinion and leave anonymous feedback. Everyone who actively participates in Officevibe surveys is made admin and can reply to feedback, so that we all help each other in solving problems and issues as they arise.
 
+#### Letter Of Interdependence
+
+We believe with the idea that we are part of a system and, when it is required, others inside the company help others. We think we are all inter-connected. If you were thinking you can maintain a mate why don't you send a message in any form declaring that you can help and saying how you will improve. Don't forget to declare your commitment also in a timeframe.
```

Comment:
We believe  that we are part of a system and, when it is required, people inside the company help each other: we believe that we are all interconnected. If you think that you could help out a teammate, you can send them a message in any form declaring that you wish help and explaining how this will make both of you improve. Don't forget to declare your commitment in a set timeframe.